README.md

```diff
@@ -1,36 +1,69 @@
 # ahk-resize-windows
+
 > Resizes and moves windows like in dwm and KDE
 
-## Usage
+Standard modkey is meta(windows key)
+
+## Features
+
 ### Resize
-Press and hold the modkey(Standard is alt) and press right click to resize
+
+Resizes the window under the mouse from the closest corner to the mouse.
+
+Modkey + Right click.
 
 ### Move
-Press and hold the modkey(Standard is alt) and press left click to move
-## Options
-### Modkey
-To change the modkey Change the string on the fifth line
-example using control: 
-```ahk
-; Options
-modKey := "ctrl"
-minimizeOnMaximized := true
-```
-[Keyname refrence](https://www.autohotkey.com/docs/v2/KeyList.htm)
-### Maximize behaviour
-Decides if resizing and moving should happen on maximized windows
 
-To change the behavior change the option on the sixth line
-example turning it off
+Move the window under the mouse.
+
+Modkey + Left click.
+
+### Fullscreen snaping
```

Comment:
typo: should be snapping

It also works for maximized windows, so write something about that